op-node/rollup/derive/fuzz_parsers_test.go

```diff
@@ -99,6 +99,35 @@ func FuzzL1InfoEcotoneRoundTrip(f *testing.F) {
 	})
 }
 
+// FuzzL1InfoKromaMPTRoundTrip checks that our Kroma MPT encoder round trips properly
+func FuzzL1InfoKromaMPTRoundTrip(f *testing.F) {
+	f.Fuzz(func(t *testing.T, number, time uint64, baseFee, blobBaseFee, hash []byte, seqNumber uint64, baseFeeScalar, blobBaseFeeScalar uint32, validatorRewardScalar []byte) {
```

Comment:
don't we need to remove validatorRewardScalar?

Reply:
Yes, that's right, marshaling - unmarshaling behavior, that's why the test called, I'll update it